experiments/run-big-bench.py

```diff
@@ -288,6 +337,13 @@ def parse_arguments() -> argparse.Namespace:
         description="Benchmark Flatnav on Big ANN datasets."
     )
 
+    parser.add_argument(
+        "--index-type",
+        required=False,
+        default="flatnav",
```

Comment:
Nit: Required defaults to False so that line is not necessary. You anyway are passing in a default value.